lib/trento/settings/analytics.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Trento.Settings.AnalyticsSettings do
+  @moduledoc """
+  Settings related to analytics.
+  """
+
+  use Ecto.Schema
+  use Trento.Support.Ecto.STI, sti_identifier: :installation_settings
+
+  import Ecto.Changeset
+
+  @type t :: %__MODULE__{}
+
+  @primary_key {:id, :binary_id, autogenerate: true}
+  schema "settings" do
+    field :analytics_optin, :boolean
+    sti_fields()
+  end
+
+  @spec changeset(t() | Ecto.Changeset.t(), map) :: Ecto.Changeset.t()
+  def changeset(analytics_settings, attrs) do
+    analytics_settings
+    |> cast(attrs, [:analytics_optin])
```

Comment:
Lets change this to:
```
analytics_settings
|> cast(attrs, [:optin])
|> validate_required([:optin]) # test this just in case, to see if it fails when false is giving, i hope it doesn't
|> sti_changes()
|> unique_constraint(:type)
```

This way, we validate all the required fields are coming and we only have one type of inherited table (you shouldn't be able to have 2, but just in case)